srcs/frontend/static/js/components/Navbar.js

```diff
@@ -8,81 +9,115 @@ export default class extends Abstract {
-											${i18next.t("navbar.home")}
-										</a>
-									</li>
-								</ul>
-							</li>
-						</ul>
-					</div>
+    return `
+        <nav class="navbar navbar-expand-lg fixed-top bg-body-tertiary">
+            <div class="container position-relative">
+                <a class="navbar-brand" href="/" data-link>
+                    <img src="/static/images/logo-42.png" style="height: 40px;"/>
+                </a>
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                    <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+                        <li class="nav-item">
+                            <a class="nav-link active" aria-current="page" href="/" data-link>
+                                ${i18next.t("navbar.home")}
+                            </a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="/pong" data-link>
+                                ${i18next.t("navbar.pong")}
+                            </a>
+                        </li>
+                        <li class="nav-item dropdown">
+                            <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+                                ${i18next.t("navbar.dashboard")}
+                            </a>
+                            <ul class="dropdown-menu">
+                                <li>
+                                    <a class="dropdown-item" href="/dashboard/individual/1" data-link>
+                                        ${i18next.t("navbar.individualDashboard")}
+                                    </a>
+                                </li>
+                                <li>
+                                    <a class="dropdown-item" href="/dashboard/general" data-link>
+                                        ${i18next.t("navbar.generalDashboard")}
+                                    </a>
+                                </li>
+                            </ul>
+                        </li>
+                        ${userManagementLinks}
+                        ${logoutLink}
+                    </ul>
+                </div>
+                ${await this.languageToggle.getHtml()}
+            </div>
+        </nav>
+    `;
+}
 
-					${await this.languageToggle.getHtml()}
-				</div>
-			</nav>
-		`;
-  }
+async refreshNavbar() {
+    const navbarContainer = document.getElementById('navbar');
+    if (navbarContainer) {
+        navbarContainer.innerHTML = await this.getHtml();
+        if (isLoggedIn()) {
+            setTimeout(() => {
```

Comment:
@pollysh why are you using a timeout here?

Reply:
I added it as part of debugging because the navbar wasn't updating properly after the signing in. I thought that maybe the DOM had been updated before attaching event listeners to new elements. I forgot to remove it after fixing the issue. Thanks for pointing this out!